unit2/README.md

```diff
@@ -8,46 +8,40 @@ Here are the steps for this unit:
 
 - Make sure you've [signed up for this course](https://huggingface.us17.list-manage.com/subscribe?u=7f57e683fa28b51bfc493d048&id=ef963b4162) so that you can be notified when new material is released
 - Read through the material below for an overview of the key ideas of this unit
-- Check out the _**FINE TUNING AND GUIDANCE NOTEBOOK TODO LINK**_ to fine-tune an existing diffusion model on a new dataset using the 🤗 Diffusers library
```

Comment:
I removed the references to conditioning for now - let me know if they should be re-instated

Reply:
I've done a small illustrative notebook and will have some extra info on conditioning to add to the readme later today.